Dockerfile.deb-stable

```diff
@@ -0,0 +1,14 @@
+FROM debian:stable
```

Comment:
Was there something special about libc 2.13?  Jessie was released in 2015, so is pretty old :)  Maybe it would be worth another Dockerfile that tests a very basic build against oldstable?